docs/source/contributors.rst

```diff
@@ -0,0 +1,67 @@
+How to Contribute!
+==================
+
```

Comment:
Needs to link to contribution guidelines at https://github.com/trustbloc/community/blob/master/CONTRIBUTING.md